MRN-Code/coinstac
# Provide for single-shot FreeSurfer computation

- **Problem:** Aspects of the UI and _client-core_ are hard-coded for [MRN-Code/decentralized-laplacian-ridge-regression](https://github.com/MRN-Code/decentralized-laplacian-ridge-regression)
- **Solution:** Un-hard-code that stuff. Update tests.
- **Testing:**
  1. Fetch and check out the _feature/single-shot-regression_ branch locally
  2. Run `npm install` in _packages/coinstac-common_
  3. Clean local disk state
  4. Build a client
  5. Boot client, create a consortium, and run the the single-shot algo

Closes #63.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/173590139517418/203258386302639)
